website/docs/intro.md

```diff
@@ -1,5 +1,13 @@
 ---
-id: intro
-title: Velox Documentation
+id: roadmap
+title: Roadmap
 slug: /
 ---
+- Add support for DECIMAL type.
```

Comment:
Maybe this should be in some In progress section ?
Also we probably need some more verbiage before abruptly going over to the items in the roadmap ?